room.go

```diff
@@ -60,14 +61,15 @@ type Room struct {
 }
 
 // CreateRoom can be used to update callbacks before calling Join
-func CreateRoom() *Room {
+func CreateRoom(callback *RoomCallback) *Room {
```

Comment:
@davidzhao agree explicit parameter is better that most case user should pass a callback here, but can also pass a nil callback if he don't care any events (just connect and publish a data msg).

Reply:
sounds like a plan!